testsuite/tools.py

```diff
@@ -15,10 +15,10 @@
 
 _tr = {
     "echo_api": "echo-api+ssl",
-    "httpbin": "httpbin+ssl",
-    "httpbin_nossl": "httpbin",
+    "httpbin": "mockserver+ssl",
+    "httpbin_nossl": "mockserver",
     "httpbin_go": "go-httpbin+ssl",
-    "httpbin_service": "httpbin+svc",
+    "httpbin_service": "echo-api+svc:9292",
```

Comment:
Why change it to echo-api when it is supposed to be an httpbin?

Reply:
https://github.com/3scale-qe/3scale-tests/pull/105/commits/761d8c7bb5a6368e40c77c0682d8673bb8807891